hxinsight-extension/src/main/java/org/alfresco/hxi_connector/hxi_extension/service/HxInsightClient.java

```diff
@@ -82,8 +82,10 @@ public List<Agent> getAgents()
     {
         try
         {
+            String url = String.format("%s?source=%s", config.getAgentUrl(), applicationInfoProvider.getSourceId());
```

Comment:
I mistyped the parameter name in the ticket.
```suggestion
            String url = String.format("%s?sourceId=%s", config.getAgentUrl(), applicationInfoProvider.getSourceId());
```

Reply:
fixed